.spotbugs/spotbugs-exclude.xml

```diff
@@ -16,5 +31,10 @@
   <Match>
     <Class name="~io\.fabric8\.kubernetes\.api\.model\.(Doneable).+(\$.*)?" />
   </Match>
+  <Match>
+    <Class name="~io\.strimzi\.systemtest\..+(\$.*)?" />
```

Comment:
Why is this OK?

Reply:
It was my understanding that we did not check the systemtests with findbugs so we do not want it with spotbugs neither.

Reply:
But _why_ do we think that's OK? Do we write better code in the system tests than in the production code? Is buggy systemtest code more acceptable? I don't think either of those arguments applies. 

So how many potential problems does spotbugs find there? If it's too many to reasonably fix in this PR we should open an issue about enabling it.

Reply:
There are 13 issues. I can fix them in this PR.